src/actinia_core/rest/base/resource_base.py

```diff
@@ -183,7 +183,9 @@ def __init__(self, resource_id=None, iteration=None, post_url=None):
 
         # Put API information in the response for later accounting
         kwargs = {
-            "endpoint": request.endpoint,
+            # For depricated location endpoints remove "_locations" from
+            # endpoint class name
+            "endpoint": request.endpoint.replace("_locations", ""),
```

Comment:
And here? I guess for the same reason :upside_down_face:

Reply:
yes

Reply:
Commented above

Reply:
and above